english/happy-robot/happy-robot.md

```diff
@@ -2,3 +2,7 @@ A happy robot helps people.
 
 It was a dark and stormy night.  A scream is heard.  A happy robot, let's name him "George," wakes up to the sound.
 "Someone needs my help!"
+
+He jumps up, but before he rushes out the door he...
+a) Grabs a [weapon](robot-help/robot-weapon.md), or
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.